install/devsds/lib/opensds.sh

```diff
@@ -84,7 +84,7 @@ osds::opensds::install(){
     export OPENSDS_AUTH_STRATEGY=$OPENSDS_AUTH_STRATEGY
     export OPENSDS_ENDPOINT=http://localhost:50040
     build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "block"}'
-    build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "file", "provisioningProperties":{"ioConnectivity": {"accessProtocol": "NFS"},"DataStorage":{"StorageAccessCapability":["Read","Write","Execute"]}}}'
+    build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "file", "provisioningProperties":{"ioConnectivity": {},"DataStorage":{}}}'
```

Comment:
Personally I prefer to add the accessProtocol and accessCapabilities explicitly.

Reply:
Ask @PravinRanjan10